dsymbol/src/dsymbol/symbol.d

```diff
@@ -432,6 +432,8 @@ struct DSymbol
 	/// Protection level for this symbol
 	IdType protection;
 
+	/// Arbitrary user data
+	void* userData;
```

Comment:
:-1: we can't know what's in here, we can't deallocate, it's just unsafe to handle. Don't use `void*`.